packages/web/src/components/gcds-card/gcds-card.tsx

```diff
@@ -109,7 +127,13 @@ export class GcdsCard {
 
     this.updateLang();
 
+    this.validateCardTitle();
+    this.validateHref();
     this.validateBadge();
+
+    if (this.error.includes('href') || this.error.includes('cardTitle')) {
+      console.error('gcds-card: Error rendering, please check required properties \'card-title\' or \'href\'');
```

Comment:
One last thing: What's the difference between showing `console.error` in a required validation check vs doing a `throw new Error()` like they show on stencils docs? https://stenciljs.com/docs/properties#prop-validation

Reply:
In my tests if I use `throw new Error()`, it will cause the page to not load fully and not continue rendering other web components. Using `console.error` just has the message in the console but everything else renders fine.

Reply:
Okay thanks for checking, yeah we don't want to do that